src/lib/components/Drawer.react.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 
 import React, { Component } from 'react';
-import { Drawer as MuiDrawer } from '@material-ui/core/Drawer';
+import MuiDrawer from '@material-ui/core/Drawer';
```

Comment:
Thank you.

Reply:
No problem, this took me forever to figure out for a different component